src/solver/hydro/daily/Test.c

```diff
@@ -44,6 +44,7 @@ main(int argc, char** argv)
     NbReservoirs = 100;
```

Comment:
This file is a ghost : it's part of the git repo, but it's not in the **cmake** system (never compiled, or scanned when searching strings over the whole Visual studio solution).
It calls the **H2O_J_EcrireJeuDeDonneesLineaireAuFormatMPS** function, which is not called anywhere else.
It only prints an **mps** file (probably for visual check).
We should consider compiling it and do something with it, or remove it otherwise.

Reply:
Good idea, I'll remove these parts of the code.